spec/augeas_parser_spec.rb

```diff
@@ -44,8 +52,8 @@
       example_tree = CFA::AugeasTree.new
       example_tree["invalid"] = "test"
 
-      msg = /Augeas parsing\/serializing error/
-      expect { subject.serialize(example_tree) }.to raise_error(msg)
+      msg = /Augeas serializing error: .* at \/etc\/sudoers::/
```

Comment:
@jreidinger This shows a weakness, the line and column do not make much sense for serializing, maybe we could point to the data in the tree??? But anyway, overall it is an improvement.